argus/backend/models/web.py

```diff
@@ -180,6 +180,7 @@ class ArgusTestRunComment(Model):
     test_run_id = columns.UUID(required=True, index=True)
     user_id = columns.UUID(required=True, index=True)
     release_id = columns.UUID(required=True, index=True)
+    test_id = columns.UUID(required=True, index=True)
```

Comment:
@k0machi I'll be asking basic questions to understand more your PR's. Please be patient :)

So, the first one, is what's a difference between `test_run_id` and `test_id`?
Also, why comment.id is needed?  Couldn't we use test_run_id as partition key and posted_at clustering key

Reply:
The table for  comments is... weird, I'm not sure why I made it like this back then and it'll need a rework at some point - you are correct  that `id` is completely extraneous and `test_run_id` should be the partition key. `test_id` is the id of the test entity that hosts runs, i.e. `longevity-10gb-3h` for `longevity-10gb-3h#3`, it is needed to later be able to make the API call to post comment.

Reply:
So, `test_id` is like a bucket containing runs of a test. And `test_run_id` matches actual `test_id` in SCT?

As to refactor, we'll see if needed later.

Reply:
Yes, that is correct. `test_id` in sct is what in argus mostly called `run_id` with some exceptions like this one (because this is a very old table)